Userland/Libraries/LibPDF/DocumentParser.h

```diff
@@ -10,6 +10,10 @@
 
 namespace PDF {
 
+struct Version {
+    int major, minor;
```

Comment:
One decl per line please (and more nitty, we generally initialize primitives in struct/class decls)